doc/safety/safety_requirements.rst

```diff
@@ -0,0 +1,113 @@
+.. _safety_requirements:
+
+Safety Requirements
+###################
+
+Introduction
+************
+The safety committee drive the effort to gather requirements to reflect the **actual** state of the
+implementation in requirements form, because of the route 3s approach of the projects safety
+effort. It is **NOT** the intention to create a new set of requirements to add them into the project
+to request new features.
+
+The requirements are gathered in the separate repository:
+`Requirement repository
+<https://github.com/zephyrproject-rtos/reqmgmt>`__
+
+
+Guidelines
+**********
+Following are the guidelines for the requirements repository and what the safety committee
+expects when requirements are added to the repository
+
+Scope
+=====
+The scope of the requirements are the **KERNEL** functionalities.
+
+Consistency
+===========
+Consistency across all requirements. The language and choice of words shall be consistent.
+(See: `Syntax`_)
+
+Levels of requirements in the repository
+========================================
+* System Requirements:
+
+  System requirements describe the behaviour of the Zephyr RTOS (= the system here).
+  They describe the functionality of the Zephyr RTOS of a very high level point of view,
+  without going into details of the functionality itself.
+  The target of the system requirements is to get an overview of the currently implemented features
+  of the Zephyr RTOS.
+  In other words a person writing these requirements usually has some knowledge of the Zephyr RTOS
+  Project as the requirements are specific to an RTOS.
+
+* Software Requirements:
+
+  Software requirements refine the requirements of the system level at a more granular level so
+  that each requirement can be tested.
+  These requirements define the specific actions the feature shall be able to execute and the
+  behavior of the feature.
+
+Requirement UID (Unique identifier) Handling
+============================================
+For the requirements management the tool `strictDoc
+<https://strictdoc.readthedocs.io/en/stable/>`__
+is used, therefore the UID (Unique identifier) handling shall be handled by the tool and this can be achived by Following
+handling:
+
+#. Don't add a requirement UID filed for new requirements
```

Comment:
```suggestion
#. Don't fill in the requirement UID when creating new requirements
```

Reply:
In this case the complete filed don't need to be added for a requirement,